### ISSUE TITLE
VPS Client Login with SSH

### ISSUE THREAD

**Issue by DVSB** [hypervm-ng/hypervm-ng#5]
I have installed HyperVM/openvz on my server and created 5 VPSs.

Some of my clients are running HyperVMNG on their VPSs and it seems the loop device is not active on the VPS, so they can't use securetmp script. 

OS on main server and VPS is centos6.9. 

I'll appreciate any advice to activate loop device on VPS.

**Comment by slrslr** [comment]
I thought HyperVM-NG is for creating virtual servers and virtual servers can not be created from within OpenVZ virtual server only from within dedicated physical server or a KVM. I googled and found there are tutorials regarding loop on OpenVZ, check that and update if you found the solution? Google phrasse: openvz loop dev

**Comment by DVSB** [comment]
Yes we have exactly with this VPS we try via ssh to access the clients. But what does not work. Error is the root passwords are not accepted.
